indico/modules/events/editing/client/js/editing/timeline/FileManager/util.js

```diff
@@ -48,33 +54,45 @@ async function uploadFile(file, url, onUploadProgress) {
   }
 }
 
+export async function uploadAnExistingFile(url, file) {
```

Comment:
Just `uploadExistingFile`? Having articles in function names is rather uncommon and sounds a bit weird

Reply:
Sounds good